src/components/battle/BattleComponent.vue

```diff
@@ -18,7 +18,7 @@
             <BattleChat v-if="!isOfflineBattle" :battle="battle" />
         </div>
         <div class="right-col flex-col gap-md">
-            <MapPreview
+            <BattleMapPreview
```

Comment:
I renamed this component because I was initially planning to make some kind of `SimpleMapPreview` component that had less canvas/startpos behavior. ATM I'm not sure if that's really necessary, so I'm happy to revert this name change.

Reply:
I think I'd rather keep the component flexible for now, and as you've used it in non-battle pages, I'll revert the name